examples/storage.vy

```diff
@@ -0,0 +1,10 @@
+stored_data: uint256
+
+@public
+def set(new_value : uint256):
+    self.stored_data = new_value
+
+@public
+@constant
+def get() -> uint256:
+    return self.stored_data
```

Comment:
@benjyz In Python you want to ensure that your script ends with newline, I guess in Vyper also 😉